plugin/pkg/admission/noderestriction/admission.go

```diff
@@ -171,6 +175,11 @@ func (p *Plugin) Admit(ctx context.Context, a admission.Attributes, o admission.
 	case resourceSliceResource:
 		return p.admitResourceSlice(nodeName, a)
 
+	case csrResource:
+		if !p.kubeletCSRAdmissionValidationDisabled {
+			return p.admitCSR(nodeName, a)
+		}
+		fallthrough
```

Comment:
nit: avoid the double-negative and use of `fallthrough` by returning early directly from this case if CSR validation is disabled:

```go
		if p.kubeletCSRAdmissionValidationDisabled {
			return nil
		}
		return p.admitCSR(nodeName, a)
```